CHANGES.rst

```diff
@@ -39,6 +39,8 @@ Bug Fixes
 Cubeviz
 ^^^^^^^
 
+- Fixed linking of data to allow contour over-plotting. [#1154]
```

Comment:
```suggestion
- Fixed linking of data to allow contour over-plotting for moment map. [#1154]
```

Reply:
It's not just for moment maps though, it does not work in main using cubes. For example, contours of VAR over FLUX in a single viewer is not possible in main, but it is in this PR.

Reply:
oh ok